client/signup/steps/plans/tabbed-plans.js

```diff
@@ -0,0 +1,986 @@
+/**
+ * External dependencies
+ */
+import {
+	applyTestFiltersToPlansList,
+	findPlansKeys,
+	getPlan as getPlanFromKey,
+	getYearlyPlanByMonthly,
+	isMonthly,
+	TERM_MONTHLY,
+} from '@automattic/calypso-products';
+import { Button } from '@automattic/components';
+import formatCurrency from '@automattic/format-currency';
+import styled from '@emotion/styled';
+import {
+	Icon,
+	image,
+	pages,
+	lifesaver,
+	media,
+	store,
+	currencyDollar,
+	trendingUp,
+} from '@wordpress/icons';
+import classNames from 'classnames';
+import { useTranslate } from 'i18n-calypso';
+import PropTypes from 'prop-types';
+import React, { useEffect, useState } from 'react';
+import { connect } from 'react-redux';
+import InfoPopover from 'calypso/components/info-popover';
+import { getCurrentUserCurrencyCode } from 'calypso/state/currency-code/selectors';
+import { getPlan, getPlanBySlug, getPlanRawPrice } from 'calypso/state/plans/selectors';
+
+/**
+ * Internal dependencies
+ */
+import './tabbed-plans-style.scss';
+
+function SharedFeatures( {
+	className = '',
+	selectedTab,
+	setShowFeatures,
+	sharedFeatures,
+	showFeatures,
+	translate,
+} ) {
+	const isMobileView = className.includes( 'mobile' );
+	const toggleFeaturesString = showFeatures
+		? translate( 'Hide all features' )
+		: translate( 'Show all features' );
+	const show = showFeatures || ! isMobileView;
+
+	return (
+		<div className={ `tabbed-plans__shared-features-grid ${ className }` }>
+			{ isMobileView && (
+				<button onClick={ () => setShowFeatures( ! showFeatures ) }>
+					{ toggleFeaturesString }
+				</button>
+			) }
+			<SharedFeatureHeader>
+				{ selectedTab === 'Professional'
+					? translate( 'All plans include' )
+					: translate( 'Personal plan also includes' ) }
+			</SharedFeatureHeader>
+			{ show &&
+				sharedFeatures[ selectedTab ].map( ( item, index ) => (
+					<SharedFeature key={ `sharedFeature${ index }` }>
+						<Icon icon={ item.icon } size={ 24 } />
+						<span>{ item.description }</span>
+					</SharedFeature>
+				) ) }
+		</div>
+	);
+}
+function TabbedPlans( { currencyCode, onUpgradeClick, planProperties } ) {
+	const translate = useTranslate();
+	const tabList = [ 'Limited', 'Professional' ];
+
+	const featureComparisonData = {
+		Professional: [
+			{
+				featureName: translate(
+					'Plugins are like apps for your site. They add new functionality and features to expand your site. There are over 58,000+ WordPress plugins available for anything you need.'
+				),
+				tooltip: translate( 'feature explanation' ),
+				planOne: { included: false, copy: translate( 'Not included' ), mobileCopy: null },
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'Not Included:' )` *ES Score: 9*